pkg/rootless/rootless_linux.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 	"os/exec"
 	gosignal "os/signal"
+	"os/user"
```

Comment:
Add line after

Reply:
why do we need an extra line here?  I thought we use it only to divide between std library and other packages